pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,186 @@
+              minimum: 0
+              description: Light intensity measured in lumens.
+            sentAt:
+              type: string
+              format: date-time
+              description: Date and time when the message was sent.`}
+</CodeBlock>
+
+Let's break it down into pieces:
+
+<CodeBlock>
+{`asyncapi: '2.5.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'`}
+</CodeBlock>
+
+- The `asyncapi` field indicates you use the AsyncAPI version 2.5.0.
+
+- The `info` field holds information about the Streetlights API. Here, the title, version, description and license were defined.
+
+Moving on, let's talk about the `servers` section.
+<CodeBlock>
+{`servers:
+  mosquitto:
+    url: mqtt://test.mosquitto.org
+    protocol: mqtt`}
+</CodeBlock> 
+
+In this section, you point to the Eclipse Mosquitto message broker. The `url` point to a real instance of the broker [hosted by the Mosquitto community](https://test.mosquitto.org/) and the `protocol` as MQTT. If you do not want to use the test instance, you can spin up your own broker locally with `docker run -it -p 1883:1883 eclipse-mosquitto:1.5`. But remember to change `url` to `mqtt://localhost`
+
+Now lets move on to the `channels` section. This section is used to describe the event names your API will be publishing and/or subscribing to.
+
+<CodeBlock>
+{`channels:
+  light/measured:
+    publish:
+      summary: Inform about environmental lighting conditions for a particular streetlight.
+      operationId: onLightMeasured`}
+</CodeBlock>
+
+In this example, `light/measured` is the channel name the Streetlight API will `subscribe` to (i.e, to interact with the Streetlight API you `publish` to the broker). The `operationId` property, describes what is the name of function or method that takes care of this functionality in the generated code.
+
+Next is the `payload` property which is used to understand how the event should look like when publishing to that channel:
+
+<CodeBlock>
+{`      payload:
+        type: object
+        properties:
+          id:
+            type: integer
+            minimum: 0
+            description: Id of the streetlight.
+          lumens:
+            type: integer
+            minimum: 0
+            description: Light intensity measured in lumens.
+          sentAt:
+            type: string
+            format: date-time
+            description: Date and time when the message was sent.`}
+</CodeBlock>
+
+The `payload` property defines the content of the event using AsyncAPI schemas. It means that your event payload should contain an `id` and a `lumens` property —which are integers bigger than zero—, and a `sentAt` property that should be a string containing a date and time.
+
+> JSON Schema Draft 07 is 100% compatible with AsyncAPI schemas.
```

Comment:
```suggestion
> JSON Schema Draft 07 is 100% compatible with AsyncAPI schemas. You can also use other standards to describe payload schema, like, for example [Avro](https://github.com/asyncapi/avro-schema-parser#usage).
```

I do not have a better link :( something for the future to remember is to add a document that explains how to use Avro, so we can link to official docs, not some GH repo

Reply:
@alequetzalli should I write this note in the `Remember` block?

Reply:
@derberg do you mean I should use Avro's official docs instead of the GH repo link you inserted?

Reply:
repo is better as there are examples on using Avro with AsyncAPI

Reply:
Yes, writing that as a `remember` block is perfect @Annysah 😄